Aseprite.js

```diff
@@ -349,6 +402,7 @@ class Aseprite {
           }) }
       }),
       palette: this.palette,
+      tileset: this.tileset,
```

Comment:
This should be `tilesets: this.tilesets,`

Reply:
Fixed in b11bcb1.